quit/helpers.py

```diff
@@ -119,3 +121,126 @@ def isAbsoluteUri(uri):
         return True
     else:
         return False
+
+
+def rewrite_graphs(parsed_query, default_graphs, named_graphs, sparql):
```

Comment:
There is not much in common for `rewrite_graphs` for update queries and for select queries. please slit these methods into methods for update and for select

Reply:
Done